src/form/FormInput.js

```diff
@@ -149,13 +146,12 @@ const styles = StyleSheet.create({
     ...Platform.select({
       android: {
         minHeight: 46,
-        width: width - 30,
       },
       ios: {
         minHeight: 36,
-        width: width,
       },
     }),
+    width: '100%',
```

Comment:
@binoy14 Not necessarily, but what other options do we have? Right now the width of the formInputs are calculated based on the screen width. So if they're nested inside a parent thats smaller then it will overflow the parent. If you have another way we could go around it, I'm all for it